resources/views/layouts/default.blade.php

```diff
@@ -101,6 +101,34 @@
 
             <header class="main-header">
 
+                <div class="row" style="background: #3c8dbd;padding: 10px;">
+                    <div class="col-md-6"></div>
+                    <div class="col-md-6">
+                        <div class="row">
+                            <div class="col-md-3">
+                                <a class="btn bg-teal" style="width: 100%" href="#">
+                                    <img height="25px" src="https://cdn-ilcenip.nitrocdn.com/MFOJuBWDJGWtSXlHPEYkvnOeIlEJMLPx/assets/images/optimized/rev-3220738/60hertzenergy.com/wp-content/uploads/elementor/thumbs/60-hertz-logo-pg5l3j7g1n69due7haggwl466f0bxga0k61ryq1b0g.png" />
```

Comment:
Many Snipe-IT installs cannot reach the outside world. Pulling in other images from other sites is not viable here.